tox.ini

```diff
@@ -21,80 +24,121 @@ extras =
     test
 
 commands =
-    pytest --cov=src --cov-report=xml --html=reports/pytest/report-{envname}.html --doctest-glob=*.rst --self-contained-html {posargs}
+    pytest --cov=src --cov-report=xml --html=reports/pytest/report-{envname}.html --self-contained-html {posargs}
     pytest --doctest-modules src/RestrictedPython/compile.py {posargs}
 
 setenv =
   COVERAGE_FILE=.coverage.{envname}
 
 deps =
+    -cconstraints.txt
     pytest-cov
     pytest-remove-stale-bytecode
     pytest-html
 
 [testenv:py27-datetime]
 basepython = python2.7
 deps =
+    -cconstraints.txt
     {[testenv]deps}
     DateTime
 
 [testenv:py36-datetime]
 basepython = python3.6
 deps =
+    -cconstraints.txt
     {[testenv]deps}
     DateTime
 
 [testenv:py27-rp3]
 basepython = python2.7
 commands =
-    coverage run {envbindir}/zope-testrunner --path=src/RestrictedPython --all -v {posargs}
+    coverage run {envbindir}/zope-testrunner --path=src/RestrictedPython --all {posargs}
```

Comment:
`-v` helps to have the same level of verbosity on zope.testrunner runs as purest runs have, so I'd like to keep it.

Reply:
I will re add this